spec/models/embedded_ansible_worker/runner_spec.rb

```diff
@@ -99,7 +99,13 @@
         runner.instance_variable_set(:@job_data_retention, ::Settings.embedded_ansible.job_data_retention_days)
       end
 
+      after do
```

Comment:
Why do we need this block? Doesn't the `runner` get re-evaluated for each example? It's defined in a `let`

Reply:
Ah, good point... I could have sworn they were class methods but clearly not... I'll verify but yeah, I think the runner should be new for each example.